src/services/mediathek.ts

```
@@ -464,15 +588,21 @@ async function applyRulesetFilters(
           matchInfo = await matchesSeasonAndEpisode(item, ruleset);
           break;
         case "ItemTitleIncludes" as MatchingStrategy:
-          matchInfo = await matchesItemTitleIncludes(item, ruleset);
+          matchInfo = await matchesItemTitleIncludes(item, ruleset, threshold);
           break;
         case "ItemTitleExact" as MatchingStrategy:
-          matchInfo = await matchesItemTitleExact(item, ruleset);
+          // For strict strategy, require higher threshold
+          const exactThreshold =
+            matchingSettings.strategy === "strict" ? Math.max(threshold, 0.95) : threshold;
+          matchInfo = await matchesItemTitleExact(item, ruleset, exactThreshold);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)